src/components/spacings/spacings.percy.js

```diff
@@ -0,0 +1,225 @@
+  display: inline-flex;
+  align-items: stretch;
+  height: 100px;
+`;
+
+const Item = styled.div`
+  background-color: ${props => props.backgroundColor};
+  height: ${props => (props.alignItems === 'stretch' ? 'auto' : props.height)};
+  width: 100px;
+`;
+
+const Scale = styled.div`
+  align-self: center;
+  width: 75px;
+  text-align: center;
+`;
+
+const InsetColorWrapper = styled.div`
+  display: inline-block;
+  background-color: #ff5b5b;
+  height: 100px;
+  width: 100px;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  > * {
+    flex-grow: 1;
+    display: flex;
+    align-items: stretch;
+  }
+`;
+
+const InsetSquishColorWrapper = styled.div`
+  background-color: #ffb15c;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  border-radius: 4px;
+  > * {
+    flex-grow: 1;
+    display: flex;
+    align-items: stretch;
+  }
+`;
+
+const Button = styled.div`
+  flex: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: white;
+  text-align: center;
+`;
+
+const Square = styled.div`
+  flex: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: white;
+  text-align: center;
+`;
+
+const StackColorWrapper = styled.div`
+  background-color: #d4e0ec;
+  width: 100px;
+  text-align: center;
+`;
+
+const sizes = [
+  { name: 'xs', pixels: '4px' },
+  { name: 's', pixels: '8px' },
+  { name: 'm', pixels: '16px' },
+  { name: 'l', pixels: '24px' },
+  { name: 'xl', pixels: '32px' },
+];
+
+const insetSquishSizes = [
+  { name: 's', pixels: '4px x 8px' },
+  { name: 'm', pixels: '8px x 16px' },
+  { name: 'l', pixels: '16px x 32px' },
+];
+
+const flexProps = ['stretch', 'flexStart', 'flexEnd', 'center'];
+const exampleHeights = ['50px', '60px', '76px', '40px', '66px'];
+
+const StackExample = ({ alignItems }) => (
+  <Spacings.Inline scale="s">
+    {sizes.map(size => (
+      <StackColorWrapper key={size.name}>
+        <Spacings.Inset scale="m">
+          <Text.Subheadline elementType="h4">
+            {size.name.toUpperCase()}
+            <Text.Detail>{size.pixels}</Text.Detail>
+          </Text.Subheadline>
+        </Spacings.Inset>
+        <Spacings.Stack scale={size.name} alignItems={alignItems}>
+          {exampleHeights.map(height => (
+            <Item
+              key={height}
+              backgroundColor="#2d68a0"
+              height={height}
+              alignItems={alignItems}
+            />
+          ))}
+        </Spacings.Stack>
+      </StackColorWrapper>
+    ))}
+  </Spacings.Inline>
+);
+
+StackExample.displayName = 'StackExample';
+StackExample.propTypes = {
+  alignItems: PropTypes.oneOf(flexProps),
+};
+
+const InlineExample = ({ alignItems }) => (
+  <Stack>
+    {sizes.map(size => (
+      <Row key={size.name}>
+        <InlineColorWrapper>
+          <Scale>
+            <Spacings.Inset scale="s" alignItems="center">
+              <Text.Subheadline elementType="h4">
```

Comment:
Something to be aware of is that by reusing components in other vrt's it is possible that these "unrelated" tests break when the used components change. Not too big of a deal as long as we don't overdo it.